packages/labs/react/src/test/create-component_test.tsx

```diff
@@ -55,6 +57,14 @@ class BasicElement extends ReactiveElement {
   @property({type: Boolean})
   disabled = false;
 
+  // override a react reserved property
+  @property({type: Boolean})
+  ref = false;
+
+  // override a react reserved property
+  @property({type: Boolean})
```

Comment:
Maybe note in comment here or in the assertion below that we only expect a warning for `ref` and not for `localName` (hence assertion of `length === 1`), since we don't warn on overrides of HTMLElement properties that React treats specially.

Reply:
done!